resources/teamcity/includes/tc-actions.inc.sh

```diff
@@ -58,3 +58,55 @@ linux_unit_tests_action() {
   builder_echo end unit_tests success "Finished running unit tests"
 }
 
+web_install_dependencies_action() {
```

Comment:
This is specific to linux dev platform, but the name does not say that

Reply:
done